org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractMalcolmTest.java

```diff
@@ -30,9 +30,20 @@
 import org.eclipse.scanning.api.malcolm.event.MalcolmEventBean;
 import org.eclipse.scanning.api.malcolm.message.MalcolmMessage;
 import org.eclipse.scanning.api.malcolm.models.MapMalcolmModel;
+import org.eclipse.scanning.malcolm.core.MalcolmDevice;
+import org.eclipse.scanning.test.malcolm.device.MockedMalcolmService;
+import org.junit.Ignore;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * TODO: JIRA ticket DAQ-1004: all the concrete tests subclasses of this class have been marked
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)